lib/cache.js

```diff
@@ -0,0 +1,64 @@
+const redis = require('redis');
+const redisLru = require('redis-lru');
+const lru = require('lru-cache');
+
+const config = require('./config');
+const logger = require('./logger');
+
+let cacheType;
+let cache;
+
+module.exports.init = function(callback) {
+    if (config.redisUrl) {
+        cacheType = 'redis';
+        const client = redis.createClient({
+            url: config.redisUrl,
+        });
+        client.on('error', (err) => logger.error(err));
+        // This max is completely arbitrary
+        cache = redisLru(client, 200);
```

Comment:
Seems low? Make it a `config` option with a much higher default? 300 students * 20 questions * 5 submissions = 30,000, so maybe 100,000 would be a good limit? At 10k per cached render that's still only 1 GB.

Reply:
Should definitely be configurable - I want to hold off on landing #1163 before making config changes.

Reply:
Ack